repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/services.yaml

```diff
@@ -632,6 +632,274 @@
+            jcr:primaryType: hippostd:html
+          jcr:primaryType: website:section
+          website:headinglevel: Main heading
+          website:numberedList: false
+          website:title: Needs a section
+          website:type: ''
+        /website:sections[2]:
+          /website:html:
+            hippostd:content: <p>for testing purposes</p>
+            jcr:primaryType: hippostd:html
+          jcr:primaryType: website:section
+          website:headinglevel: Main heading
+          website:numberedList: false
+          website:title: Needs another section
+          website:type: ''
+        /website:seosummary:
+          hippostd:content: ''
+          jcr:primaryType: hippostd:html
+        /website:summary:
+          hippostd:content: <p>testing page only</p>
+          jcr:primaryType: hippostd:html
+        /website:userContact:
+          /website:userContactnotes:
+            hippostd:content: ''
+            jcr:primaryType: hippostd:html
+          jcr:primaryType: website:userContact
+          website:userContactPhone: ''
+          website:userContactemail: ''
+          website:userContactname: ''
+          website:userContactprimarycontact: false
+        /website:userNeed:
+          jcr:primaryType: website:userNeed
+          website:asA: ''
+          website:iNeedto: ''
+          website:soThat: ''
+        /website:userNeedandcontact:
+          /website:briefDescription:
+            hippostd:content: ''
+            jcr:primaryType: hippostd:html
+          /website:teampicker:
+            hippo:docbase: cafebabe-cafe-babe-cafe-babecafebabe
+            jcr:primaryType: hippo:mirror
+          jcr:primaryType: website:userNeedandcontact
+          website:portfolioCode: ''
+          website:responsibleTeam: ''
+          website:userNeedismet:
+          - ''
+        common:searchRank: 6
+        common:searchable: true
+        hippo:availability:
+        - live
+        hippostd:state: published
+        hippostdpubwf:createdBy: admin
+        hippostdpubwf:creationDate: 2020-10-12T15:43:23.959Z
+        hippostdpubwf:lastModificationDate: 2020-10-12T15:46:29.003Z
+        hippostdpubwf:lastModifiedBy: admin
+        hippostdpubwf:publicationDate: 2020-10-12T15:46:34.579Z
+        hippotranslation:id: bcc4582e-15d5-42a8-8667-2a3f5e491f93
+        hippotranslation:locale: en
+        jcr:mixinTypes:
+        - hippotaxonomy:classifiable
+        - mix:referenceable
+        jcr:primaryType: website:service
+        website:gossid: 0
+        website:htmlCode: ''
+        website:metadata: []
+        website:navigationcontroller: extraWideColumn
+        website:rawMetadata:
+        - ''
+        website:shortsummary: ''
+        website:title: Lesley testing service pages
+      hippo:versionHistory: 04dc0b6e-9376-48ce-bc3c-3f6e21801220
+      jcr:mixinTypes:
+      - hippo:versionInfo
+      - mix:referenceable
+      jcr:primaryType: hippo:handle
```

Comment:
Same comment as the content in general.ymal. Did you mean to commit this content?

Reply:
Hi, if its development I didnt think these got published, I am not too sure how this happened, perhaps you could explain and how i could prevent it happening in the future?